glean-core/src/upload/mod.rs

```diff
@@ -29,6 +29,9 @@ mod result;
 // The maximum size in bytes a ping body may have to be eligible for upload.
 const PING_BODY_MAX_SIZE: usize = 1024 * 1024; // 1 MB
 
+// The maximum size in byte the pending pings directory may have on disk.
+const PENDING_PINGS_DIRECTORY_QUOTA: usize = 15 * 1024 * 1024; // 15 MB
```

Comment:
@Dexterp37 On desktop we keep 15 MB in the directory, but set it to 1MB for mobile gecko (Fennec).
15 MB seems like _a lot_ here, should we adjust that?

Reply:
I gave this suggestion on my initial comment for this PR, but one thing we could do is make this into a config option. Each binding could define the quota that makes sense for it.

Reply:
@badboy when we added that different mobile quota on Desktop/Fennec, the world was very different place, SDs had lower space, etc. etc. - From a Firefox Desktop POV, 15MB means about 45days worth of pings. In the mobile world, that's twice as much (at least!). I'd advocate for using one single quota, make things simpler. Possibly lower this to 10MB.

Reply:
I agree I want a single quota (for now, maybe configurable in the future -> separate bug).
Sticking with 10 MB for now sounds reasonable.